index.html

```diff
@@ -77,9 +77,8 @@
       <section class="works padding">
```

Comment:
- [ ] The work cards in the section are not using the same margin and padding as in the Figma design, kindly consider adding some margin around the cards and using the right amount of padding in the cards so they look exactly like the provided design, this will show your ability to duplicate and create a page from a giving design 

|Your design|Figma design|
|-|-|
|![Screenshot_20220901_214216](https://user-images.githubusercontent.com/25227022/188009232-b1e6d63d-b290-4260-946e-8a8a37bf6e91.png)|![Screenshot_20220901_214232](https://user-images.githubusercontent.com/25227022/188009250-e513fde0-6736-4888-91b7-12a4e5b1f211.png)|

Reply:
Good job following the advice of the previous reviewer